tutorials/migrating-rails-app-from-heroku-to-google-appengine.md

```diff
@@ -255,43 +251,40 @@ them as environment variables into your app's runtime environment.
 App Engine provides two ways to set environment variables, depending on how
 you'd like to manage and secure the content. First, you can set them directly
 in the `app.yaml` file by creating an `env_variables` section, which App Engine
-will inject into your app's runtime environment. For example:
+injects into your app's runtime environment. For example:
 
     env_variables:
       DATABASE_USER: postgres
       RAILS_MAX_THREADS: "5"
 
-This is useful for simple configuration. However, you will typically want to
+This is useful for simple configuration. However, you typically want to
 save your `app.yaml` config file to source control, so it is not a good place
 for secrets such as your database password or your `SECRET_KEY_BASE`. This
 kind of sensitive data can instead be managed securely using the
 [Runtime Configuration Service](https://cloud.google.com/deployment-manager/runtime-configurator/),
 a Google service similar to the Heroku config system.
 
-In your app, you probably need to manage at least two secret values: the
+In your app, you likely need to manage at least two secret values: the
 `DATABASE_PASSWORD`, and the `SECRET_KEY_BASE`. Let's store them in Runtime
-Configuration.
-
-1.  Enable the Runtime Config API in the Cloud Console
```

Comment:
Again, where do we tell people to enable these APIs?

Reply:
Done.